src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,34 +15,33 @@
 package com.google.sps.servlets;
```

Comment:
Skipping this file because I think it is part of PR#20? I can re-review once that PR is merged.